infra/label_evaluator_step_functions/lambdas/utils/tracing.py

```
@@ -242,6 +242,57 @@ def get_child_headers(self) -> Optional[dict]:
+                "metadata": {
+                    "langsmith_trace_id": self.trace_id,
+                    "langsmith_parent_run_id": (
+                        self.run_tree.id if self.run_tree else None
+                    ),
+                },
+                "run_id": self.run_tree.id if self.run_tree else None,
+                # Pass headers for trace linking
+                "configurable": {
+                    "langsmith_headers": headers,
+                },
+            }
```

Ruff (0.14.13)

Consider moving this statement to an `else` block

(TRY300)